docs/documenting/general-style-guide.md

```
@@ -420,3 +420,37 @@ Use "caution" to advise against actions that may cause data loss, unintended beh
+Use this text box when documenting features in Labs:
+
+```liquid
+{% labs %}
+Requires the **Feature Name** Labs preview feature. Enable it at {% my labs title="**Settings** > **System** > **Labs**" %}.
+{% endlabs %}
+```
+
+#### Example YAML
+
+Use this text box to provide interactive syntax highlighting for some elements.
+To see the example below rendered, see [Automation: sync a ceiling fan speed to the ceiling light ](https://www.home-assistant.io/triggers/light.brightness_changed/#automation-sync-a-ceiling-fan-speed-to-the-ceiling-light).
```

markdownlint-cli2 (0.22.0)

[warning] Spaces inside link text

(MD039, no-space-in-links)